dnf-behave-tests/features/steps/cmd.py

```diff
@@ -122,6 +120,9 @@ def step_impl(context):
 def step_impl(context):
     context.dnf._set("disable_plugins", False)
 
+@behave.given("I set dnf command to \"{command}\"")
+def step_impl(context, command):
+    context.dnf._set("dnf_command", command)
```

Comment:
Why not just do `context.dnf_command = command`? the `_set` method just looks like a roundabout way to do the same...